netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -513,12 +531,16 @@ const MapContainer = () => {
   return (
     <div>
       <ErrorBoundary>
```

Comment:
thanks for this @FaithDaka , Is this code here related to the initial zoom level in our maps?

I initially suggested that we have our map focus on the whole of Africa on first load.